app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,165 @@
+      ],
+      'CT' => [
+        # Daily, 4:00 am. to 6:30 am. ET.
+        { cron: '0 4 * * *', duration: 90 },
+        # Sunday 6:00 am. to 9:30 am. ET
+        { cron: '0 6 * * Mon', duration: 3.5 * 60 },
+      ],
+      'DC' => [
+        # Daily, Midnight to 6 am. ET.
+        { cron: '0 0 * * *', duration: 6 * 60 },
+      ],
+      'DE' => [
+        # Daily, Midnight to 5 am. ET.
+        { cron: '0 0 * * *', duration: 5 * 60 },
+      ],
+      'FL' => [
+        # Sunday 7:00 am. to 12:00 pm. ET
+        { cron: '0 7 * * Sun', duration: 5 * 60 },
+      ],
+      'IA' => [
+        # "Daily system resets, normally at 4:45 am. to 5:15 am ET."
+        { cron: '45 4 * * *', duration: 30 },
+      ],
+      'IN' => [
+        # Sunday morning maintenance from 6 am. to 10 am. ET.
+        { cron: '0 6 * * Sun', duration: 4 * 60 },
+      ],
+      'IL' => [
+        { cron: '30 2 * * *', duration: 2.5 * 60 }, # Daily, 2:30 am. to 5 am. ET.
+      ],
+      'KY' => [
+        # Daily maintenance from 2:50 am. to 6:40 am. ET
+        { cron: '50 2 * * *', duration: 230 },
+      ],
+      'MA' => [
+        # Daily maintenance from 6 am. to 6:15 am. ET.
+        { cron: '0 6 * * *', duration: 15 },
+        # Wednesday 7 am. to 7:30 am. ET.
+        { cron: '0 7 * * Wed', duration: 30 },
+        # Saturday 10:00 pm. to Sunday 10:00 am
+        { cron: '0 22 * * Sat', duration: 12 * 60 },
+        # First Friday of each month: 12 to 6 am. ET.
+        { cron: '0 0 * * Fri#1', duration: 6 * 60 },
+      ],
+      'MD' => [
+        # Daily maintenance from 3 am. to 3:15 am. ET.
+        { cron: '0 3 * * *', duration: 15 },
+        # Sunday maintenance may occur from 6 am. to 10 am. ET.
+        { cron: '0 6 * * Sun', duration: 4 * 60 },
+      ],
+      'MI' => [
+        # Daily maintenance from 9 pm. to 9:15 pm. ET.
+        { cron: '0 21 * * *', duration: 15 },
+      ],
+      'MO' => [
+        # Daily maintenance from 2 am. to 4:30 am. ...
+        { cron: '0 2 * * *', duration: 2.5 * 60 },
+        # ... from 6:30 am to 6:45 am ...
+        { cron: '30 6 * * *', duration: 15 },
+        # ... and 8:30 am. to 8:35 am ET.
+        { cron: '30 8 * * *', duration: 5 },
+        #  Sundays from 9 am. to 10:30 am. ET...
+        { cron: '0 9 * * Sun', duration: 90 },
+        # ...and 5 am to 5:45 am ET on 2nd Sunday of month.
+        { cron: '0 5 * * Sun#2', duration: 45 },
+      ],
+      'NC' => [
+        # Daily, Midnight to 7:00 am. ET.
+        { cron: '0 0 * * *', duration: 7 * 60 },
+        # Sundays from 5am. till Noon
+        { cron: '0 5 * * Sun', duration: 7 * 60 },
+      ],
+      # NM: "Sunday mornings." (not modeling; too vague)
+      'NY' => [
+        # Sunday maintenance 8 pm. to 9 pm. ET.
+        { cron: '0 20 * * Sun', duration: 60 },
+      ],
+      'PA' => [
+        # Sunday maintenance may occur, often between 5:30 am. & 7:00 am. ET
+        { cron: '30 5 * * Sun', duration: 90 },
```

Comment:
Note the imprecision in the comment (which comes from an AAMVA doc) -- I want to call out that we're modeling _likely_ maintenance windows. It's possible for maintenance to occur outside of these times, and likely that some states will be available during the stated maintenance windows.